src/lib/auth.js

```diff
@@ -16,7 +16,7 @@ exports.setup = function(express, app, config) {
       done(null, obj)
     })
 
-    var callbackUrl = 'http://' + config.host + '/auth/google/callback'
+    var callbackUrl = config.host + '/auth/google/callback'
```

Comment:
The url scheme is now part of the host url (see https://github.com/guardian/elk-stack/pull/61/files#diff-fea7794cc929f6e1c59c3329bf062533R539)